plugins/woocommerce/tests/legacy/unit-tests/rest-api/Tests/Version2/system-status.php

```diff
@@ -59,17 +71,13 @@ public function test_get_system_status_info_without_permission() {
 	public function test_get_system_status_info_returns_root_properties() {
 		$this->skip_on_php_8_1();
 
-		wp_set_current_user( $this->user );
-		$response = $this->server->dispatch( new WP_REST_Request( 'GET', '/wc/v2/system_status' ) );
-		$data     = $response->get_data();
-
-		$this->assertArrayHasKey( 'environment', $data );
-		$this->assertArrayHasKey( 'database', $data );
-		$this->assertArrayHasKey( 'active_plugins', $data );
-		$this->assertArrayHasKey( 'theme', $data );
-		$this->assertArrayHasKey( 'settings', $data );
-		$this->assertArrayHasKey( 'security', $data );
-		$this->assertArrayHasKey( 'pages', $data );
+		$this->assertArrayHasKey( 'environment', $this->get_system_status_data() );
```

Comment:
Maybe rather than calling `$this->get_system_status_data()` in every assertion we could call it once, store the output, and then assert against that?